python/raf/amp/type_hints.py

```diff
@@ -122,6 +122,7 @@ def _gen(args, ret_type, amp_dtype):
 register_op_cast_rule("raf.op.arange", generic_cast(False, 3))
 register_op_cast_rule("raf.op.exp", generic_cast(False, 1))
 register_op_cast_rule("raf.op.power", generic_cast(False, 1))
+register_op_cast_rule("raf.op.reciprocal", generic_cast(False, 1))
```

Comment:
Does fp16 hurt the accuracy?

Reply:
Checked https://pytorch.org/docs/stable/amp.html#autocast-op-reference
Reciprocal op will not be autocasted to fp16.